subprojects/inline/src/test/java/org/mockitoinline/StaticMockTest.java

```diff
@@ -20,6 +23,9 @@
 
 public final class StaticMockTest {
 
+    @Rule
+    public ExpectedException exception = ExpectedException.none();
```

Comment:
Rather than this rule, let's keep it consistent with the rest of the codebase and use `assertj`. Then we can wrap the statement that we expect to fail, without letting it fail the whole method.

Reply:
Ah yes makes sense. I filed #2510 for us to clean up `ExpectedException` and will clean some up today. Thanks for the fix!